presto-main/src/main/java/io/prestosql/operator/scalar/MathFunctions.java

```diff
@@ -607,6 +607,50 @@ public static long random(@SqlType(StandardTypes.BIGINT) long value)
         return ThreadLocalRandom.current().nextLong(value);
     }
 
+    @Description("a pseudo-random between start and stop (exclusive)")
```

Comment:
To be consistent with existing function. Other places are also the same. 
```suggestion
    @Description("A pseudo-random number between start and stop (exclusive)")
```

Reply:
Of course @ebyhr! I'll resolve conflicts of MathFunctions. About negative random values, I got it wrong the comment of electrum. I'll revert this change. 
Thank you and sorry for the confusion caused.